spec/features/user_signup_functionality_spec.rb

```diff
@@ -89,13 +91,19 @@
       click_button "Create"
       expect(page).to have_content("Email has already been taken")
     end
-
-    xit "can not create an account if the user is already signed in" do
-      visit "/register"
-      # what should the current path be? Should they even see the sign
-      # up page or should see that page but the sign up funtionality not work
-      # if they try to sign up?
-      expect(page).to have_content("Please sign out before creating a new account")
+  end
+  context "user is already Logged in" do
+    before(:each) do
+      visit "/"
+      click_link("Login")
+      fill_in("Email", with: 'jeff@gmail.com')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.